configs/sabaki/README.md

```diff
@@ -0,0 +1,72 @@
+# How to set up sabaki
+
+You can install Sabaki from https://sabaki.yichuanshen.de/.
+If you are on MacOS, you can install using homebrew with the command
+`brew install sabaki`.
+
+## Bot setup
+0. First build the C++ katago executable.
+   To do this, 
+   you'll want to run the following commands in both the
+   `engines/KataGo-custom/cpp`
+   and
+   `engines/KataGo-raw/cpp`
+   directories.
+   ```
+   cmake . -DUSE_BACKEND=OPENCL -DUSE_TCMALLOC=1
+   make -j
+   ```
+   Adjust the backend and whether to use tcmalloc as needed.
+1. Open up Sabaki preferences by pressing command comma (⌘,).
```

Comment:
someone should check what the command is on Linux, but I'm also on macOS so I don't know

or we can just write that the user should go to Preferences in the top-level drop-down menu

Reply:
In Linux it's File->Preferences

Reply:
Updated with instructions for Linux.